src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        Machine[] machinesArr = new Machine[]{truck,bulldozer,excavator};
+
+        for (Machine i : machinesArr) {
```

Comment:
"i" is not informative, improve naming  + it usually is being used in for loop as an index

Reply:
But it is index for loop =)

Reply:
I can change it but I thought it would be right  to keep the index as usual

Reply:
it's not a index, it's object. 
Index has type "int" 
your "i" has type "Machine"
you can't use i.startWork() in the loop (for int i =0; i < machinesArr.length; i++)

Reply:
Understood that  and changed it in last commit